### PR TITLE
@types/react(-dom)@18

### PR DESCRIPTION
**Pull request by nstepien**

### CODE REVIEW

File: src/formatters/ToggleGroupFormatter.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export function ToggleGroupFormatter<R, SR>({
       tabIndex={tabIndex}
       onKeyDown={handleKeyDown}
     >
-      {groupKey}
+      {groupKey as string}
```
Comment (nstepien) [inline src/formatters/ToggleGroupFormatter.tsx:48]: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56210#issuecomment-1092021240

File: website/demos/ContextMenu.tsx [hunk 2]
```diff
@@ -129,6 +129,7 @@ function rowKeyGetter(row: Row) {
 
 function RowRenderer(props: RowRendererProps<Row>) {
   return (
+    // @ts-expect-error
     <ContextMenuTrigger id="grid-context-menu" collect={() => ({ rowIdx: props.rowIdx })}>
```
Comment (nstepien) [inline website/demos/ContextMenu.tsx:133]: `children` isn't implicitly typed anymore.

Comment (amanmahajan7) [inline website/demos/ContextMenu.tsx:133]: May be we can remove ContextMenu library and add a basic popup for examples.

**Review by amanmahajan7 — APPROVED** [review]

File: website/demos/ContextMenu.tsx (hunk 2 above)
Comment (amanmahajan7) [reply website/demos/ContextMenu.tsx:133]: I will check

Comment (nstepien) [reply website/demos/ContextMenu.tsx:133]: Preferably yes, the library isn't maintained anymore.

**Pull request merged**